packages/babel-generator/test/fixtures/typescript/types-with-comments-retainLines/output.js

```diff
@@ -0,0 +1,23 @@
+type T = U.
+/* 1 */
+C /* 2 */[
+/* 3 */
+0];
+
+type T2 = U.
+/* 1 */
+C /* 2 */[];
+
+
```

Comment:
There seems to be a bug here.

Reply:
Yes, the `/* 3 */` comment should be printed _somewhere_. It's a bug that we already have regardless of this PR.